### DISCUSSION TITLE
Store media files on Google Cloud Storage

### DISCUSSION THREAD

**Discussion by mariia-artamonova** [ant-media/Ant-Media-Server#5944] [FAQs] [answered]
My company is using Google Cloud Platform, and it would like to use Google Cloud Storage (GCS) as main storage instead of AWS S3.

I have googled and search for questions and answers on this community and I did not find any answer or tutorial, or plugin to integrate to GCS.

Is there anyway to record and store lived stream media to GCS?

**Comment by yashtandon113** [comment, marked as answer]
@mariia-artamonova 
Thanks for posting the question.

Please check out the following document for AMS integration with Google Cloud Storage (GCS).
https://antmedia.io/docs/guides/recording-live-streams/s3-integration-http-forwarding/#record-streams-to-google-cloud-storage

Also make sure that [HTTP forwarding](https://antmedia.io/docs/guides/recording-live-streams/s3-integration-http-forwarding/#http-forwarding) is enabled so VOD can be played directly using the AMS URL.
